src/items.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+readonly class Item
```

Comment:
You should rename this to be more specific, Trainer would be better. 

Very minor thing, since Item does make sense now - always think about what could potentially happen in the future with any given project. Fair enough in this case nothing is going to happen after this week, but pretend it is :D 

If we expanded this app into other collections, Item suddenly doesn't make sense any more